plugin/src/main/java/net/pl3x/map/configuration/Advanced.java

```diff
@@ -636,24 +641,24 @@ Setting a color to black (#000000) will make it invisible.""")
         put(Blocks.MANGROVE_WALL_SIGN, 0x763631);
         put(Blocks.MANGROVE_WOOD, 0x544329);
         put(Blocks.MEDIUM_AMETHYST_BUD, 0xA37CCC);
-        put(Blocks.MELON, 0x6D901E);
         put(Blocks.MELON_STEM, 0x9B9B9B);
+        put(Blocks.MELON, 0x6D901E);
         put(Blocks.MOSS_BLOCK, 0x596E2D);
         put(Blocks.MOSS_CARPET, 0x596E2D);
-        put(Blocks.MOSSY_COBBLESTONE, 0x6D765E);
         put(Blocks.MOSSY_COBBLESTONE_SLAB, 0x6D765E);
         put(Blocks.MOSSY_COBBLESTONE_STAIRS, 0x6D765E);
         put(Blocks.MOSSY_COBBLESTONE_WALL, 0x6D765E);
+        put(Blocks.MOSSY_COBBLESTONE, 0x6D765E);
         put(Blocks.MOSSY_STONE_BRICK_SLAB, 0x74796A);
         put(Blocks.MOSSY_STONE_BRICK_STAIRS, 0x74796A);
         put(Blocks.MOSSY_STONE_BRICK_WALL, 0x74796A);
         put(Blocks.MOSSY_STONE_BRICKS, 0x74796A);
-        put(Blocks.MOVING_PISTON, 0x5A5A5A);
-        put(Blocks.MUD, 0x3C3A3D);
+        put(Blocks.MOVING_PISTON, 0x000000);
```

Comment:
I dont think the moving piston is actually in game anywhere, but I colored it same as piston, just in case.